### DISCUSSION TITLE
Disable VSCode testing feature?

### DISCUSSION THREAD

**Discussion by jeffpeterson** [Shopify/ruby-lsp#2780] [Q&A]
How do I disable the VSCode Ruby LSP testing feature? It doesn't appear to work very well and is causing errors to show up in my editor.

**Comment by andyw8** [comment]
There are several features related to testing, can you describe which is giving problems?

**Reply by jeffpeterson** [reply]
Sorry, by "testing", I mean the "Testing" panel in VSCode. https://code.visualstudio.com/api/extension-guides/testing

I'm using https://marketplace.visualstudio.com/items?itemName=connorshea.vscode-ruby-test-adapter for testing, and I'd like to prevent ruby-lsp from also adding tests to the Testing panel.
